### DISCUSSION TITLE
How to get scaling factor between projections?

### DISCUSSION THREAD

**Discussion by woutware** [DotSpatial/DotSpatial#1548] [Q&A]
I've used Reproject.ReprojectPoints(xy, z, projectionInfoSource, projectionInfoTarget, 0, 1); successfully to reproject points, but how do I get the scale factor (or matrix?) from source to target ProjectionInfo? I was thinking method Reproject.ReprojectAffine could be helpful, but I have no clue how to use it, and I don't see examples or unit test code for it in https://github.com/DotSpatial/DotSpatial/blob/master/Source/Tests/DotSpatial.Projections.Tests/ReprojectTests.cs.

**Comment by sindizzy** [comment]
What do you need it for?

**Reply by woutware** [reply]
This is to know how dimensions scale in X and Y direction. Specifically it is to be used for scaling text height from an AutoCAD drawing e.g. in some local source coordinate system, scaled/transformed to a map in WGS84 coordinate system (longitude/latitude in degrees). So the goal is the find the text size expressed in degrees (either longitude or latitude).
